modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/WebServerLauncher.scala

```diff
@@ -50,7 +53,11 @@ object WebServerLauncher extends ServerApp with LogInitialization {
       val host = cfg.require[String]("web-server.host")
       val port = cfg.require[Int]("web-server.port")
       val devMode = cfg.require[String]("mode")
-      WebServerConfiguration(host, port, devMode.equalsIgnoreCase("dev"))
+      val keystore = cfg.lookup[String]("web-server.tls.keyStore")
+      val keystorePwd = cfg.lookup[String]("web-server.tls.keyStorePwd")
+      val certPwd = cfg.lookup[String]("web-server.tls.certPwd")
+      val sslConfig = (keystore |@| keystorePwd |@| certPwd)(SSLConfig.apply)
```

Comment:
Nice. Of all the different syntax to write Applicatives, this looks the least ugly to me.